app/src/main/java/com/example/randomuserapplication/util/Constants.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.randomuserapplication.util
+
+object Constants {
+    const val BASE_URL = "https://randomuser.me/"
+    const val END_POINT = "api/?results=500&inc=name,picture,location,nat,gender,dob,login&noinfo"
+}
```

Comment:
why is your endpoint in this form? why is this `inc=name,picture,location,nat,gender,dob,login&noinfo` needed?

Reply:
In the documentation, they said if you wanted specific information, do it like this. You want me to change it? To extract all the information?
![api](https://github.com/Obehi234/Random-User-Generator/assets/48982222/1ccee3b5-1d11-455e-8d6a-ace22513f2ba)